### PR TITLE
Updated docs on MPICH_RANK_REORDER

### PR DESCRIPTION
**Pull request by aturner-epcc**
Final part of scheduler section update

### CODE REVIEW

File: user-guide/scheduler.rst [hunk 1]
```diff
@@ -608,7 +608,7 @@ specify other types of MPI task placement. For example, setting it to
 
     module load xthi
     export MPICH_RANK_REORDER_METHOD=0
-    srun --nodes=8 --tasks-per-node=2 xthi
+    srun --cpu-bind=core xthi
```
Comment (juanfrh) [inline user-guide/scheduler.rst]: It should be _--cpu-bind=cores_

File: user-guide/scheduler.rst [hunk 2]
```diff
@@ -600,24 +609,33 @@ specify other types of MPI task placement. For example, setting it to
...
+  salloc --nodes=8 --tasks-per-node=2 --cpus-per-task=1 --time=0:10:0 --account=t01
+
+  salloc: Granted job allocation 24236
+  salloc: Waiting for resource configuration
+  salloc: Nodes cn13 are ready for job
+
+  module load xthi
+  export OMP_NUM_THREADS=1
+  export MPICH_RANK_REORDER_METHOD=0
+  srun --cpu-bind=core xthi
```
Comment (juanfrh) [inline user-guide/scheduler.rst:621]: It should be _--cpu-bind=cores_

File: user-guide/scheduler.rst (hunk 1 above)
Comment (aturner-epcc) [reply user-guide/scheduler.rst]: Fixed in https://github.com/ARCHER2-HPC/archer2-docs/pull/48/commits/e5b33cb6bab8e9f6412ff58249049a5d0cab48bf

File: user-guide/scheduler.rst (hunk 2 above)
Comment (aturner-epcc) [reply user-guide/scheduler.rst]: Fixed in https://github.com/ARCHER2-HPC/archer2-docs/pull/48/commits/e5b33cb6bab8e9f6412ff58249049a5d0cab48bf

**Comment by mbareford** [comment]
Should it not be "--nodes=4" for both the default and round robin cases?

**Comment by aturner-epcc** [comment]
@mbareford I do not think so, you can see the nid numbers go from 0 to 7 indicating 8 nodes in use.

**Comment by mbareford** [comment]
@aturner-epcc Apologies, I was looking at a previous version of that page.

**Pull request merged**